### ISSUE TITLE
TV-Type: Every input after the third becomes it's own device in homekit

### ISSUE THREAD

**Issue by Ting3l** [kiwi-cam/homebridge-broadlink-rm#357]
When adding more than three inputs to a device of the type tv, the additional inputs after the third one become their own device in homekit and are not shown unter the "inputs" in the tv-device itself.

**Comment by masterfordev** [comment]
Post your config. I have 99 inputs on my tv (channels) and its working fine.

**Comment by Ting3l** [comment]
Config:
`{
            "platform": "BroadlinkRM",
            "name": "Broadlink RM",
            "hideScanFrequencyButton": true,
            "hideLearnButton": false,
            "hideWelcomeMessage": false,
            "logLevel": "info",
            "accessories": [
                {
                    "name": "Flimmerkiste",
                    "type": "tv",
                    "data": {
                        "off": "...",
                        "on": "...",
                        "remote": {
                            "select": [
                                {
                                    "data": "..."
                                },
                                {
                                    "data": "..."
                                }
                            ],
                            "arrowUp": [
                                {
                                    "data": "..."
                                },
                                {
                                    "data": "..."
                                }
                            ],
                            "arrowDown": [
                                {
                                    "data": "..."
                                },
                                {
                                    "data": "..."
                                }
                            ],
                            "arrowLeft": [
                                {
                                    "data": "..."
                                },
                                {
                                    "data": "..."
                                }
                            ],
                            "arrowRight": [
                                {
                                    "data": "..."
                                },
                                {
                                    "data": "..."
                                }
                            ],
                            "back": [
                                {
                                    "data": "..."
                                },
                                {
                                    "data": "..."
                                }
                            ],
                            "exit": "",
                            "info": [
                                {
                                    "data": "..."
                                },
                                {
                                    "data": "..."
                                }
                            ],
                            "playPause": [
                                {
                                    "data": "..."
                                },
                                {
                                    "data": "..."
                                }
                            ],
                            "rewind": "",
                            "fastForward": "",
                            "nextTrack": "",
                            "previousTrack": ""
                        },
                        "volume": {
                            "up": [
                                {
                                    "data": "...",
                                    "pause": 0.1
                                },
                                {
                                    "data": "...",
                                    "pause": 0.1
                                },
                                {
                                    "data": "...",
                                    "pause": 0.1
                                },
                                {
                                    "data": "...",
                                    "pause": 0.1
                                },
                                {
                                    "data": "...",
                                    "pause": 0.1
                                }
                            ],
                            "down": [
                                {
                                    "data": "...",
                                    "pause": 0.1
                                },
                                {
                                    "data": "...",
                                    "pause": 0.1
                                },
                                {
                                    "data": "...",
                                    "pause": 0.1
                                },
                                {
                                    "data": "...",
                                    "pause": 0.1
                                },
                                {
                                    "data": "...",
                                    "pause": 0.1
                                }
                            ]
                        },
                        "inputs": [
                            {
                                "name": "PC",
                                "type": "hdmi",
                                "data": [
                                    {
                                        "data": "...",
                                        "pause": 4
                                    },
                                    {
                                        "data": "...",
                                        "pause": 2
                                    },
                                    {
                                        "data": "...",
                                        "pause": 1
                                    },
                                    {
                                        "data": "...",
                                        "pause": 1
                                    },
                                    {
                                        "data": "..."
                                    }
                                ]
                            },
                            {
                                "name": "Switch",
                                "type": "hdmi",
                                "data": [
                                    {
                                        "data": "...",
                                        "pause": 4
                                    },
                                    {
                                        "data": "...",
                                        "pause": 2
                                    },
                                    {
                                        "data": "...",
                                        "pause": 1
                                    },
                                    {
                                        "data": "...",
                                        "pause": 1
                                    },
                                    {
                                        "data": "...",
                                        "pause": 1
                                    },
                                    {
                                        "data": "...
                                ]
                            },
                            {
                                "name": "PS4",
                                "type": "hdmi",
                                "data": [
                                    {
                                        "data": "...",
                                        "pause": 4
                                    },
                                    {
                                        "data": "...",
                                        "pause": 2
                                    },
                                    {
                                        "data": "...",
                                        "pause": 1
                                    },
                                    {
                                        "data": "...",
                                        "pause": 1
                                    },
                                    {
                                        "data": "...",
                                        "pause": 1
                                    },
                                    {
                                        "data": "...",
                                        "pause": 1
                                    },
                                    {
                                        "data": "..."
                                    }
                                ]
                            },
                            {
                                "name": "Satelite",
                                "type": "hdmi",
                                "data": [
                                    {
                                        "data": "...",
                                        "pause": 4
                                    },
                                    {
                                        "data": "...",
                                        "pause": 2
                                    },
                                    {
                                        "data": "...",
                                        "pause": 1
                                    },
                                    {
                                        "data": "...",
                                        "pause": 1
                                    },
                                    {
                                        "data": "...",
                                        "pause": 1
                                    },
                                    {
                                        "data": "...",
                                        "pause": 1
                                    },
                                    {
                                        "data": "..."
                                    }
                                ]
                            }
                        ]
                    }
                }
            ]
        }`

I'll get back with screenshots from my iPhone, but basically it creates a subdevice for the tv named "Satelite", but only the first three inputs are shown in the tv-device.



EDIT: Deleted screenshots

**Comment by Ting3l** [comment]
![0526D4BA-B35F-4202-8528-AEE6E834A4C5](https://user-images.githubusercontent.com/16223882/124901656-7c8ab480-dfe2-11eb-832b-620fa228f0df.png)
![31D6434F-86BE-41D4-926D-454232DF86E6](https://user-images.githubusercontent.com/16223882/124901672-801e3b80-dfe2-11eb-996b-7bcc82617c19.png)
![B30B9B43-2302-4F9D-A84D-079413E686A4](https://user-images.githubusercontent.com/16223882/124901673-80b6d200-dfe2-11eb-9e06-6a160cc50b84.png)
![C0D8B33B-DDA1-4B79-9AAE-6EB9E74CEA31](https://user-images.githubusercontent.com/16223882/124901674-80b6d200-dfe2-11eb-85eb-525b24765d2b.png)

**Comment by Ting3l** [comment]
Have done a bit more testing today.
Even adding a fifth device produces the same error.
Also, I noticed the additional devices always return "on", and I cannot seem to operate them in any way.
Also changing some of the input-types from hdmi to "other" or "tuner" had no effect at all.
For now I did a workaround with a bunch of auto-off-switches bringing me to the desired input and sorting them into a seperate room in homekit, but I guess this is not intended and should be fixed? 
I don't know if this is a problem with homekit, homebridge or this plugin.
Anyways, I'm currently running:
iOS 14.6
Homebridge v1.2.5
homebridge-broadlink-rm-pro v4.4.5
Broadlink RM Mini 3
